docs/HISTORY.txt

```diff
@@ -2,10 +2,13 @@ Changelog
 =========
 
 
-1.0.8 (unreleased)
+2.0.0 (unreleased)
```

Comment:
@jone But a backwards incompatible API change is the DEFINITON of a major version: http://semver.org/
If we use `1.1.0` others will think this is a backwards compatible update and will update it without a second thought. And then things break and people get angry. 😉